src/spaces/jolt_physics_direct_space_state_3d.cpp

```diff
@@ -169,10 +169,15 @@ int32_t JoltPhysicsDirectSpaceState3D::_intersect_shape(
 	JPH::CollideShapeSettings settings;
 	settings.mMaxSeparationDistance = (float)p_margin;
 
+	if (JoltProjectSettings::use_enhanced_edge_removal()) {
+		settings.mActiveEdgeMode = JPH::EActiveEdgeMode::CollideWithAll;
+		settings.mCollectFacesMode = JPH::ECollectFacesMode::CollectFaces;
+	}
```

Comment:
@jrouwe Am I right in enabling `EActiveEdgeMode::CollideWithAll` when using `InternalEdgeRemovingCollector`? I assume this new collector makes the active edge detection redundant?

(The results I've seen from some preliminary testing are great by the way.)

Reply:
It should be redundant, but I think it is still faster to leave the other on as well because of this early out:

https://github.com/jrouwe/JoltPhysics/blob/398f571b1a7c30be330bc6c5ce2e3aaffdc93916/Jolt/Physics/Collision/InternalEdgeRemovingCollector.h#L115

Reply:
B.t.w. you may want to put this one back to default again:

https://github.com/jrouwe/JoltPhysics/blob/398f571b1a7c30be330bc6c5ce2e3aaffdc93916/Jolt/Physics/Collision/Shape/MeshShape.h#L60

Reply:
Ah, gotcha, thank you. I went ahead and made it use a fixed threshold of 5 degrees when you enable this new enhanced internal edge removal. I also went ahead and enabled the setting by default.